browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -1842,6 +1842,42 @@ IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, DefaultNoRemoveparam) {
                 base::BindRepeating(content::FrameHasSourceUrl, frame_url)));
 }
 
+// `$removeparam` should still be activated in default blocking mode if it comes
```

Comment:
There is one test case I think we should add: same test as the one that's in the PR, but the reverse. In other words, we should test that if we have a `removeparam` in a *default list* in standard mode, it's not removed. So in that case the final URL should still have `evil` in it.

Reply:
@antonok-edm just pointed out to me that the test I was asking for [already existed](https://github.com/brave/brave-core/blob/f3a2fef05e9dc7ea6fae4a4f07d01d5d9989f9b3/browser/brave_shields/ad_block_service_browsertest.cc#L1819-L1843) prior to this PR. So it's not part of this PR, but it's covered. :+1: